src/ClassFinder.php

```diff
@@ -45,6 +45,10 @@ public function getDeclaredClasses() : Collection
             return starts_with($value, 'Illuminate'); // Ignore Illuminate Packages
         })->reject(function ($value, $key) {
             return starts_with($value, 'Symfony');
+        })->reject(function ($class) {
```

Comment:
I would actually like to map these to ReflectionClass instances up front and transform these checks to operate on that.

I only did this to change as little as possible. Would that be a worthwhile refactoring for this PR or should it be separate? (If the checks do end up living here at the end)